sycl/source/detail/device_info.hpp

```diff
@@ -18,6 +18,9 @@
 #include <sycl/detail/pi.hpp>
 #include <sycl/device.hpp>
 #include <sycl/ext/oneapi/experimental/device_architecture.hpp>
+#if (SYCL_EXT_ONEAPI_MATRIX_VERSION == 4)
```

Comment:
@dkhaldi should it be >= 4 instead of == 4 ?

Reply:
=4 should be enough.